operator-common/src/main/java/io/strimzi/operator/common/model/cruisecontrol/CruiseControlEndpoints.java

```diff
@@ -36,7 +36,12 @@ public enum CruiseControlEndpoints {
     /**
      * Remove broker
      */
-    REMOVE_BROKER("/kafkacruisecontrol/remove_broker");
+    REMOVE_BROKER("/kafkacruisecontrol/remove_broker"),
+
+    /**
+     * metrics
+     */
+    METRICS("/metrics");
```

Comment:
`metrics` => `Metrics` if we want to keep this. But I'm not sure it is worth to pull this into a production code when it is relevant only to tests.

Reply:
Removed